src/config/backpack/base.php

```diff
@@ -175,6 +175,9 @@
     // You can make sure all your URLs use this prefix by using the backpack_url() helper instead of url()
     'route_prefix' => 'admin',
 
+    // The middleware (group) used in all base routes
+    'route_middleware' => 'web',
```

Comment:
I think I'd rather call this ```web_middleware```. Since that's what it is.

Reply:
If @pxpm agrees with the change I will fix those 2 points!

Reply:
Go ahead :)